### ISSUE TITLE
Unable to determine the status of checkboxes with screen readers

### ISSUE THREAD

**Issue by ribbons** [ribbons/RadioDownloader#169]
**Original report from** [Elliott Barnes](mailto:elliottabarnes@googlemail.com) **at** 10:55:15 **on** 2012-03-30

Since the latest 0.24 update to Radio Downloader, screen readers such as NVDA are unable to determine the status of checkboxes, and as a result screen reader users don't know if they are checked or unchecked. This can become an issue if multiple episodes are selected, etc.

Here's how to produce:
1. Download and install NVDA from http://NVDA-Project.org
2. Select any programme/podcast, and open the Download window where episodes can be selected, and use up/down arrows to move between them.

Expected: NVDA should announce the name of the episode, and then the status of the checkbox (either checked or unchecked)

Result: NVDA announces the name of the episode, but not the status of the checkbox.

I'm assuming that this has something to do with the fact that multiple episodes can now in fact be selected.

---

Imported from Bug 630 in the [NerdoftheHerd.com](http://nerdoftheherd.com) Bugzilla.
